IBPSA/Fluid/CHPs/OrganicRankine/Cycle.mo

```diff
@@ -0,0 +1,494 @@
+within IBPSA.Fluid.CHPs.OrganicRankine;
+model Cycle "Organic Rankine cycle as a bottoming cycle"
+
+  extends IBPSA.Fluid.Interfaces.FourPortHeatMassExchanger(
+    final m1_flow_nominal = mHot_flow_nominal,
+    final dp1_nominal = dpHot_nominal,
+    final m2_flow_nominal = mCol_flow_nominal,
+    final dp2_nominal = dpCol_nominal,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    T1_start = max(pro.T)*2/3 + min(pro.T)*1/3,
+    T2_start = max(pro.T)*1/10 + min(pro.T)*9/10,
+    redeclare final IBPSA.Fluid.MixingVolumes.MixingVolume vol2(
+      V=m2_flow_nominal*tau2/rho2_nominal,
+      nPorts=2,
+      final prescribedHeatFlowRate=true),
+    final vol1(
+      final prescribedHeatFlowRate=true));
+
+  replaceable parameter IBPSA.Fluid.CHPs.OrganicRankine.Data.Generic pro
+    constrainedby IBPSA.Fluid.CHPs.OrganicRankine.Data.Generic
+    "Property records of the working fluid"
+    annotation(choicesAllMatching = true,Dialog(group="Cycle"));
+  parameter Modelica.Units.SI.MassFlowRate mHot_flow_nominal
+    "Nominal mass flow rate of the evaporator fluid"
+    annotation(Dialog(group="Evaporator"));
+  parameter Modelica.Units.SI.PressureDifference dpHot_nominal(
+    displayUnit="Pa")
+    "Nominal pressure drop of the hot fluid in evaporator"
+    annotation(Dialog(group="Evaporator"));
+  parameter Modelica.Units.SI.TemperatureDifference dTPinEva_set(
+    final min = 0) = 5
+    "Set evaporator pinch point temperature difference"
+    annotation(Dialog(group="Evaporator"));
+  parameter Boolean useEvaporatingPressure = false
+    "Set true to specify working fluid evaporating pressure instead of temperature"
+    annotation(Dialog(group="Evaporator"));
+  parameter Modelica.Units.SI.ThermodynamicTemperature TWorEva =
+    max(pro.T)*2/3 + min(pro.T)*1/3
```

Comment:
This is a placeholder for handling "conditional" parameters. The intention is that the user can either use $T_{wor,eva}$ or $p_{wor,eva}$ to specify the cycle. Even though only one of these two parameters is needed, Modelica would complain unless both are given a value. Hence a hopefully reasonable default value is given here.

Reply:
Makes sense. Can you document the reason for this default if users wonder? Because it looks like a physical equation, even though it is just an educated guess.

Reply:
Updated documentation in a7934de.